lib/ansible/galaxy/collection.py

```diff
@@ -615,7 +698,7 @@ def download_collections(collections, output_path, apis, validate_certs, no_deps
                 if requirement.api is None and requirement.b_path and os.path.isfile(requirement.b_path):
                     shutil.copy(requirement.b_path, to_bytes(dest_path, errors='surrogate_or_strict'))
                 elif requirement.api is None and requirement.b_path:
-                    temp_path = to_text(b_temp_path, errors='surrogate_or_string')
```

Comment:
Does this cause a failure today?

Reply:
I'm not sure, I just noticed the typo and went looking to see if 'surrogate_or_string' was actually a thing.

Reply:
If you were still curious about this, looks like is can cause a failure
```
>>> from ansible.module_utils.common.text.converters import to_text
>>> to_text(b'[\xff]', 'ascii', errors='nope')
\Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/liveuser/ansible/lib/ansible/module_utils/common/text/converters.py", line 216, in to_text
    return obj.decode(encoding, errors)
LookupError: unknown error handler name 'nope'
```